src/components/LastPlayedSong.tsx

```
@@ -0,0 +1,78 @@
+interface CurrentMusic {
+  albumArt: string;
+  title: string;
+  grandparentTitle: string;
+}
+
+const getMusicData = async (): Promise<CurrentMusic | null> => {
+  try {
+    const response = await fetch('https://rosnovsky.us/api/now-playing');
+    const data = await response.json();
+    return data[0];
+    console.log(data[0])
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)